### PR TITLE
Removes the Configuration Context Manager

### PR DESCRIPTION
**Pull request by zwoop**
This is a set of management APIs that was used by the Web UI, for creating,
modifying and deleting configuration files and entries. This is another step
towards removing the requirements for writing to configuration files.

### CODE REVIEW

**Comment by zwoop** [comment]
Note that I left these are two separate commits, intentionally, to make it somewhat more reasonable to review. Both commits are functional, compilable states.

File: lib/perl/lib/Apache/TS/AdminClient.pm [hunk 1]
```diff
@@ -30,28 +30,27 @@ use Apache::TS;
 
 # Mgmt API command constants, should track ts/mgmtapi.h
```
Comment (PSUdaemon) [inline lib/perl/lib/Apache/TS/AdminClient.pm]: I might be missing it, but I don't see a removal for the corresponding line in `ts/mgmtapi.h`

Comment (zwoop) [reply lib/perl/lib/Apache/TS/AdminClient.pm]: Yeh, it's not in ts/mgmtapi.h, it's deep in the core. Look for READ_FILE in there :-)

Comment (PSUdaemon) [reply lib/perl/lib/Apache/TS/AdminClient.pm]: We should probably update or remove that comment then?

Comment (zwoop) [reply lib/perl/lib/Apache/TS/AdminClient.pm]: Yeh, it's not correct. I got fooled by this too initially.

**Pull request merged**